priam/src/main/java/com/netflix/priam/aws/auth/S3RoleAssumptionCredential.java

```diff
@@ -80,11 +69,20 @@ public AWSCredentialsProvider getAwsCredentialProvider() {
                         // session/token expiration.
                         try {
 
+                            StsClient stsClient = StsClient.builder()
+                                    .credentialsProvider(this.cred.getAwsCredentialProvider())

```

Comment:
Remove `this.` prefix. That is only necessary when you need to disambiguate between an object property and a local method variable of the same name.